fix_output_tests.py

```
@@ -0,0 +1,10 @@
+with open("crates/logfwd-output/src/lib.rs", "r") as f:
```

Ruff (0.15.9)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)